openedx/openedx-platform
# Enable In-App purchases on edx-mobile app

## Description

Changes to enable In-App purchases on edx mobile app.
Add android_skus and ios_skus in course modes to enable purchases.
Return course modes in enrollment API.


## Supporting information

Link to Jira ticket: https://2u-internal.atlassian.net/browse/LEARNER-9205

## Testing instructions

1. Apply migrations
2. Edit ios_sku and android_sku in Course modes and ensure data persistence
3. Ensure course modes are returned in enrollments api by calling the api in browser for a logged in user. API Url: {domain}/api/mobile/v2/users/{username}/course_enrollments/

## Deadline

Rough deadline 3rd February.

## Other information

Merge to stage first so mobile team can perform testing. Merge/Deploy to prod after testing is complete.
